### ISSUE TITLE
Iraq / PRD Partners marked for deletion are still displaying

### ISSUE THREAD

**Issue by aldohunicef** [unicef/etools-issues#453]
in Trips the list of partners is even showing the ones that have been marked for deletion:

**Comment by aldohunicef** [comment]
![image](https://user-images.githubusercontent.com/26142645/27153546-e67a859c-511f-11e7-9d2a-40a5af953c5b.png)

**Comment by anusenan** [comment]
@ntrncic  please check this.

**Comment by robertavram** [comment]
@anusenan  this would have been something pulilabs should have looked at.
this issue is a frontend issue..
simple fix would be relace the url here:
`https://github.com/unicef/etools-t2f/blob/master/t2f/app-elements/api-elements/inbound-data.html#L241`
with
`url: '/api/v2/partners/?hidden=False'`

simple fix.. let us know if this needs to. be prioritized.. we can push it through the pipeline..

**Comment by anusenan** [comment]
@janpanchalingam1 please comment on this.

**Comment by janpanchalingam1** [comment]
@anusenan @robertavram yes, this is still an issue and as we have requested country offices to enter assurance activities to update their hact dashboard, this should be prioritized.

**Comment by janpanchalingam1** [comment]
Failed testing in staging. Workspace: Iraq. Partners marked for deletion showing in trips: 
![image](https://user-images.githubusercontent.com/26144740/29984129-fcdb09c0-8f26-11e7-8e61-b4d3143b05d2.png)

**Comment by janpanchalingam1** [comment]
@ntrncic this issue is in other work spaces. For example in Jordan:
![image](https://user-images.githubusercontent.com/26144740/30563548-3865c22a-9c90-11e7-8023-8ed729dd884e.png)

Are we able to fix this across workspaces?

**Comment by ShrinivasPuranik** [comment]
Week Sep 26, Needs to be fixed this week

**Comment by janpanchalingam1** [comment]
Partners marked for deletion are still showing:
![image](https://user-images.githubusercontent.com/26144740/32110300-3368ebf6-bb05-11e7-961c-e0a362bbc438.png)

![image](https://user-images.githubusercontent.com/26144740/32110331-4414e6bc-bb05-11e7-97eb-c1a222ba1c87.png)

**Comment by ntrncic** [comment]
@janpanchalingam1 as you see in the Partner Organization list there are two EAST AMMAN CHARITY. Only one is showing up in the dropdown in T2F (EAC/EAST AMMAN CHARITY) as expected. The EAC is just a short name for the partner.

**Comment by janpanchalingam1** [comment]
Thanks Nik. Passed testing.
